tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -2040,23 +2040,23 @@ declare namespace ts {
         getNonNullableType(type: Type): Type;
         getTypeArguments(type: TypeReference): readonly Type[];
         /** Note that the resulting nodes cannot be checked. */
-        typeToTypeNode(type: Type, enclosingDeclaration?: Node, flags?: NodeBuilderFlags): TypeNode | undefined;
+        typeToTypeNode(type: Type, enclosingDeclaration: Node | undefined, flags: NodeBuilderFlags | undefined): TypeNode | undefined;
```

Comment:
This would be a break if we want it for 3.9.  Any way we can avoid it?

Reply:
Easily - now that I've used it to audit our internal usages, swapping it back is pretty simple; changing it is just for safety at callsites.